src/NuoJsOptions.cpp

```diff
@@ -32,6 +33,14 @@ Options& Options::operator=(const Options& options)
     return *this;
 }
 
+bool Options::operator==(const Options& rhs) const {
+	return (rowMode == rhs.rowMode) &&
+	       (fetchSize == rhs.fetchSize) &&
+	       (isolationLevel == rhs.isolationLevel) &&
+	       (autoCommit == rhs.autoCommit) &&
+	       (readOnly == rhs.readOnly);
+}
+
```

Comment:
I'm not sure I see this being used anywhere -- is this a relic from something you were trying to do, or is there a specific purpose to this code?

Reply:
It was used as part of something I was checking, which was removed, but a working equality operator did not seem like a useless artifact to have around, so I left it in for some other future work to compare if multiple option instances are equivalent in value.